contrib/devtools/copyright_header.py

```diff
@@ -94,7 +93,6 @@ def compile_copyright_regex(copyright_style, year_style, name):
     r"Satoshi Nakamoto",
     r"The Bitcoin Core developers",
     r"BitPay Inc\.",
-    r"University of Illinois at Urbana-Champaign\.",
```

Comment:
Only found these in Python dependencies, so: 👍 

Unrelated: we could update the script to change the end year to `present` instead